### ISSUE TITLE
Remove test run classes from the project

### ISSUE THREAD

**Issue by JeremyMcCormick** [JeffersonLab/hps-java#782]
There is legacy code used during the Test Run which is almost certainly all dysfunctional, and it should be removed. We do not plan on processing or analyzing Test Run data at all. This will reduce the amount of clutter in the project and help if and when refactoring occurs.

The Test Run conditions can be left in the database in the (very unlikely) chance that they are ever needed in the future, though alternately a SQL dump could be saved with them, and they could be removed to reduce the number of tables and amount of data in the database.

Test Run detectors can be removed as they are also almost certainly broken and should not be used.

I do not feel that we need to spend the time to try and find out if Test Run data is still usable, as the basic EVIO to LCIO component was checked years ago and did not work. The situation now is almost certainly even worse in terms of this data not being loadable into the framework.

**Comment by JeremyMcCormick** [comment]
This is the obvious list of files to remove:

```
./conditions/src/main/java/org/hps/conditions/ecal/TestRunEcalConditionsConverter.java
./conditions/src/main/java/org/hps/conditions/svt/TestRunSvtChannel.java
./conditions/src/main/java/org/hps/conditions/svt/TestRunSvtConditions.java
./conditions/src/main/java/org/hps/conditions/svt/TestRunSvtConditionsConverter.java
./conditions/src/main/java/org/hps/conditions/svt/TestRunSvtDaqMapping.java
./conditions/src/main/java/org/hps/conditions/svt/TestRunSvtT0Shift.java
./conditions/src/test/java/org/hps/conditions/svt/TestRunSvtBadChannelsTest.java
./conditions/src/test/java/org/hps/conditions/svt/TestRunSvtConditionsConverterTest.java
./conditions/src/test/java/org/hps/conditions/svt/TestRunSvtDaqMappingTest.java
./detector-data/detectors/HPS-TestRun-v1/HPS-TestRun-v1.lcdd
./detector-data/detectors/HPS-TestRun-v2/HPS-TestRun-v2.lcdd
./detector-data/detectors/HPS-TestRun-v3/HPS-TestRun-v3.lcdd
./detector-data/detectors/HPS-TestRun-v3-split/HPS-TestRun-v3-split.lcdd
./detector-data/detectors/HPS-TestRun-v4/HPS-TestRun-v4.lcdd
./detector-data/detectors/HPS-TestRun-v5/HPS-TestRun-v5.lcdd
./detector-data/detectors/HPS-TestRun-v6/HPS-TestRun-v6.lcdd
./detector-data/detectors/HPS-TestRun-v7/HPS-TestRun-v7.lcdd
./detector-data/detectors/HPS-TestRun-v7-2/HPS-TestRun-v7-2.lcdd
./detector-data/detectors/HPS-TestRun-v7-3/HPS-TestRun-v7-3.lcdd
./detector-data/detectors/HPS-TestRun-v8/HPS-TestRun-v8.lcdd
./detector-data/detectors/HPS-TestRun-v8-4/HPS-TestRun-v8-4.lcdd
./detector-data/detectors/HPS-TestRun-v8-5/HPS-TestRun-v8-5.lcdd
./detector-data/detectors/HPSTestRunTracker2014-v0/HPSTestRunTracker2014-v0.lcdd
./detector-model/src/main/java/org/lcsim/detector/converter/compact/HPSTestRunTracker2014Converter.java
./detector-model/src/main/java/org/lcsim/detector/tracker/silicon/HpsTestRunSiSensor.java
./detector-model/src/main/java/org/lcsim/geometry/compact/converter/HPSTestRunTracker2014GeometryDefinition.java
./detector-model/src/main/java/org/lcsim/geometry/compact/converter/HPSTestRunTracker2014JavaBuilder.java
./detector-model/src/main/java/org/lcsim/geometry/compact/converter/HPSTestRunTracker2014LCDDBuilder.java
./detector-model/src/main/java/org/lcsim/geometry/compact/converter/lcdd/HPSTestRunTracker2014.java
./detector-model/src/main/java/org/lcsim/geometry/subdetector/HPSTestRunTracker2014.java
./detector-model/src/test/java/org/hps/detector/svt/TestRunSvtDetectorSetupTest.java
./detector-model/src/test/java/org/lcsim/detector/converter/compact/HpsTestRunSiSensorConverterTest.java
./detector-model/src/test/java/org/lcsim/geometry/compact/converter/lcdd/HPSTestRunTracker2014LCDDTest.java
./detector-model/src/test/java/org/lcsim/geometry/subdetector/HPSTestRunTracker2014Test.java
./detector-model/src/test/resources/org/lcsim/geometry/subdetector/HpsTestRunSiSensorConverterTest.xml
./detector-model/src/test/resources/org/lcsim/geometry/subdetector/HPSTestRunTracker2014.xml
./ecal-readout-sim/src/main/java/org/hps/readout/ecal/TestRunTriggerDriver.java
./evio/src/main/java/org/hps/evio/LCSimTestRunEventBuilder.java
./evio/src/main/java/org/hps/evio/TestRunReconToEvio.java
./evio/src/main/java/org/hps/evio/TestRunSvtEvioReader.java
./evio/src/main/java/org/hps/evio/TestRunTriggeredReconToEvio.java
./evio/src/main/java/org/hps/evio/TestRunTriggeredReconToLcio.java
./evio/src/test/java/org/hps/evio/LCSimTestRunEventBuilderTest.java
./evio/src/test/java/org/hps/evio/TestRunSvtEvioReaderTest.java
./integration-tests/src/test/resources/org/hps/steering/test/HPSTestRunTrackingDefaults.lcsim
./integration-tests/src/test/resources/org/hps/steering/test/TestRun2014_CondCheck.lcsim
./recon/src/main/java/org/hps/recon/particle/TestRunReconParticleDriver.java
./record-util/src/main/java/org/hps/record/triggerbank/TestRunTriggerData.java
./steering-files/src/main/resources/org/hps/steering/broken/TestRunOfflineRecon.lcsim
./steering-files/src/main/resources/org/hps/steering/broken/TestRunOfflineReconTwoTrkFilter.lcsim
./steering-files/src/main/resources/org/hps/steering/readout/legacy_drivers/TestRunNoPileup.lcsim
./steering-files/src/main/resources/org/hps/steering/readout/legacy_drivers/TestRunReadoutToEvio.lcsim
./steering-files/src/main/resources/org/hps/steering/readout/legacy_drivers/TestRunReadoutToLcio.lcsim
./steering-files/src/main/resources/org/hps/steering/readout/legacy_drivers/TestRunReadoutToLcio2014.lcsim
./tracking/src/main/resources/org/hps/recon/tracking/strategies/HPS-TestRun-135.xml
./tracking/src/main/resources/org/hps/recon/tracking/strategies/HPS-TestRun-357.xml
```

**Comment by JeremyMcCormick** [comment]
Rename this class

./evio/src/main/java/org/hps/evio/LCSimTestRunEventBuilder.java

to LCSimBaseEventBuilder, rather than removing (it is fairly important!).

**Comment by JeremyMcCormick** [comment]
Remove these entire directories:

```
./detector-data/detectors/HPS-TestRun*
```

**Comment by JeremyMcCormick** [comment]
Resolved by #871
